app/actions/views/root.js

```diff
@@ -16,9 +16,14 @@ import {markChannelAsRead, viewChannel} from 'mattermost-redux/actions/channels'
 
 export function loadConfigAndLicense(serverVersion) {
     return async (dispatch, getState) => {
-        getClientConfig()(dispatch, getState);
-        getLicenseConfig()(dispatch, getState);
-        setServerVersion(serverVersion)(dispatch, getState);
+        const config = await getClientConfig()(dispatch, getState);
```

Comment:
Is there some way to fire these off in parallel still? I don't know if this works, but something like
```
const [config, license] = await Promise.all([
    getClientConfig(),
    getLicenseConfig()
]);
```